chapter/src/main/java/com/nk/DetailsResorce.java

```diff
@@ -1,63 +1,103 @@
 package com.nk;
 
 import java.net.URI;
-import java.text.DateFormat;
-import java.util.Date;
-
+import java.net.URISyntaxException;
 import javax.ws.rs.*;
-import javax.ws.rs.core.MediaType;
-import javax.ws.rs.core.NewCookie;
-import javax.ws.rs.core.Response;
+import javax.ws.rs.core.*;
+
 
 
 @Path("users")
 public class DetailsResorce {
     DetailsRepository repo =new DetailsRepository();
+
     @POST
     @Path("signup")
     @Consumes({MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON})
     public Response userSignup (UserDetails user) {
-        System.out.println("getting called");
-        String output = "Created customer <a href=\"customers/" +
-                user.getUsername() + "\">" + user.getUsername()
-                + "</a>";
-        String lastVisit = DateFormat.getDateTimeInstance(
-                DateFormat.SHORT, DateFormat.LONG).format(new Date());
-        if(repo.signup(user)){
+        System.out.println(user.getUsername());
```

Comment:
@anandh619 help her remove `system.out.println`  and move all these to logger for debugging